build.gradle

```diff
@@ -6,9 +6,9 @@
 
 buildscript {
     ext {
-        opensearch_version = System.getProperty("opensearch.version", "2.0.0-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "2.0.0-rc1-SNAPSHOT")
```

Comment:
This will revert changes that are in the upstream/main branch. 
Let's be careful that we aren't undoing Joshua's work.